cloudbuild.yaml

```diff
@@ -1,130 +1,109 @@
-    id: 'copy-multicluster-2-config'
+- name: gcr.io/cloud-builders/gcloud
+  entrypoint: cp
+  args:
+  - /builder/home/.kube/config
+  - /builder/home/.kube/solo-kit-multicluster-2-config
+  id: 'copy-multicluster-2-config'
 
-  - name: gcr.io/cloud-builders/gcloud
-    args: ['container', 'clusters', 'get-credentials', 'solo-kit-test']
-    id: 'gcloud-solo-kit-test'
+- name: gcr.io/cloud-builders/gcloud
+  args: ['container', 'clusters', 'get-credentials', 'solo-kit-test']
+  id: 'gcloud-solo-kit-test'
 
-  # e2e-ginkgo is produced from https://github.com/solo-io/cloud-builders/e2e-ginkgo
-  # sets up redis, consul, kubectl, go with required environment variables
-  # need to use the provided entrypoint
-  - name: 'gcr.io/$PROJECT_ID/e2e-ginkgo:0.1.17'
-    dir: './gopath/src/github.com/solo-io/solo-kit'
-    env:
-      - 'PROJECT_ROOT=github.com/solo-io/solo-kit'
-      - 'CLOUDSDK_COMPUTE_ZONE=us-central1-b'
-      - 'CLOUDSDK_CONTAINER_CLUSTER=solo-kit-test'
-      - 'RUN_KUBE_TESTS=1'
-      - 'RUN_CONSUL_TESTS=1'
-      - 'RUN_VAULT_TESTS=1'
-      - 'SKIP_MOCK_GEN=1'
-      - 'CLOUD_BUILD=1'
-      - 'GOPATH=/workspace/gopath'
-      - 'ALT_CLUSTER_KUBECONFIG=/builder/home/.kube/solo-kit-multicluster-2-config'
-    args: ['-r', '-v', '-race', '-p', '-tags', 'solokit', '-compilers=10']
-    volumes: *vol
-    id: 'test'
+# e2e-ginkgo is produced from https://github.com/solo-io/cloud-builders/e2e-ginkgo
+# sets up redis, consul, kubectl, go with required environment variables
+# need to use the provided entrypoint
+- name: 'gcr.io/$PROJECT_ID/e2e-go-mod-ginkgo:0.2.1'
+  env:
+  - 'PROJECT_ROOT=github.com/solo-io/solo-kit'
+  - 'CLOUDSDK_COMPUTE_ZONE=us-central1-b'
+  - 'CLOUDSDK_CONTAINER_CLUSTER=solo-kit-test'
+  - 'RUN_KUBE_TESTS=1'
+  - 'RUN_CONSUL_TESTS=1'
+  - 'RUN_VAULT_TESTS=1'
+  - 'SKIP_MOCK_GEN=1'
+  - 'CLOUD_BUILD=1'
+  - 'ALT_CLUSTER_KUBECONFIG=/builder/home/.kube/solo-kit-multicluster-2-config'
+  args: ['-r', '-v', '-race', '-p', '-tags', 'solokit', '-compilers=2']
+  dir: *dir
+  id: 'test'
 
-  - name: 'gcr.io/$PROJECT_ID/go-make:0.1.17'
-    args: ['solo-kit-cli']
-    waitFor: ['test']
-    dir: './gopath/src/github.com/solo-io/solo-kit'
-    id: 'build-cli'
-    volumes: *vol
-    env:
-    - 'GOPATH=/workspace/gopath'
+- name: 'gcr.io/$PROJECT_ID/go-mod-make:0.2.1'
+  args: ['solo-kit-cli']
+  waitFor: ['test']
+  id: 'build-cli'
+  dir: *dir
+  env:
 
-  - name: 'gcr.io/$PROJECT_ID/go-make:0.1.17'
-    args: ['solo-kit-gen']
-    waitFor: ['test']
-    dir: './gopath/src/github.com/solo-io/solo-kit'
-    id: 'build-codegen'
-    volumes: *vol
-    env:
-    - 'GOPATH=/workspace/gopath'
+- name: 'gcr.io/$PROJECT_ID/go-mod-make:0.2.1'
+  args: ['solo-kit-gen']
+  waitFor: ['test']
+  id: 'build-codegen'
+  dir: *dir
+  env:
 
 timeout: 1500s
 tags: ['solo-kit']
 options:
   machineType: 'N1_HIGHCPU_32'
-  env:
-  - "GO111MODULE=on"
\ No newline at end of file
+  volumes:
```

Comment:
The GO111MODULE=auto by default for go 1.13, but the auto setting now activates the module-aware mode of the go command whenever the current working directory contains, or is below a directory containing, a go.mod file — even if the current directory is within GOPATH/src. This change simplifies the migration of existing code within GOPATH/src and the ongoing maintenance of module-aware packages alongside non-module-aware importers.

yay